crates/bevy_app/src/app.rs

```diff
@@ -515,7 +515,7 @@ impl App {
     /// Adds a new [State] with the given `initial` value.
```

Comment:
```suggestion
    /// Adds a new [`State`] with the given `initial` value.
```

Reply:
Ah, good call. This won't cause a clippy warning unfortunately because there's no CamelCase, which is why I missed it.